tests/integration/conftest.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for Synapse charm integration tests."""
+
+
+import json
+
+import pytest_asyncio
+from juju.model import Model
+from pytest import Config
+from pytest_operator.plugin import OpsTest
+
+
+@pytest_asyncio.fixture(scope="module", name="server_name")
+async def fixture_server_name() -> str:
+    """Return a server_name."""
+    return "my.synapse.local"
+
+
+@pytest_asyncio.fixture(scope="module", name="model")
+async def fixture_model(ops_test: OpsTest) -> Model:
+    """Return the current testing juju model."""
+    assert ops_test.model
+    return ops_test.model
+
+
+@pytest_asyncio.fixture(scope="module", name="build_charm")
```

Comment:
Should the fixture name be what is returned by the fixture rather than what it does? E.g., just call it `charm`?

Reply:
I assumed that since in [Flask](https://github.com/canonical/flask-k8s-operator/blob/main/tests/integration/conftest.py#L57) was written like that then it would be ok. But I agree on that too. I'm changing to "synase_charm".